### PR TITLE
Links to videos, articles, sites added

### PR DESCRIPTION
**Pull request by StephanStS**
@MichaIng: I did not put every link into quotes (`), we should discuss this.

Also changing some of the links to embedded videos is to be decided.

### CODE REVIEW

**Comment by MichaIng** [comment]
Legend, many thanks!

File: .wordlist.txt [hunk 1]
```diff
@@ -121,6 +122,7 @@ frontend
 fullscreen
 FuguHub
 Gbit
+german
```
Comment (MichaIng) [inline .wordlist.txt:125]: I think "German language" (always capitalised) is correct, isn't it?

File: docs/dietpi_references.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# References
+
+At this page, links to general information are referenced.  
+Links to special software packages are located at the end of the according software package description in the "Optimized Software" documentation area.
+
+## General links
+
+- Twitter: <https://twitter.com/dietpi_>
```
Comment (MichaIng) [inline docs/dietpi_references.md:8]: Let's use `https://twitter.com/DietPi_` here, it practically makes no difference but matches what you get when opening the profile page from within Twitter.

File: docs/dietpi_references.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+- Facebook: <https://www.facebook.com/groups/167403007229675>
+
+## General articles
+
+- Raspberry Pi Foundation: [DietPi | Minimal image, highly extensible](https://www.raspberrypi.org/forums/viewtopic.php?t=285096)
+- Hardkernel ODROID: [[OS] DietPi | Minimal image, highly extensible](https://forum.odroid.com/viewtopic.php?t=40392)
+- PC Welt (german language): [`Diet Pi ist erste Wahl für Headlessplatinen`](https://www.pcwelt.de/a/diet-pi-ist-erste-wahl-fuer-headlessplatinen,3452321)
```
Comment (MichaIng) [inline docs/dietpi_references.md:15]: I wanted to correct it being "DietPi" without space, but I guess it has priority to use the actual video title, right? 😄

**Review by MichaIng — CHANGES_REQUESTED** [review]

File: .wordlist.txt (hunk 1 above)
Comment (StephanStS) [reply .wordlist.txt:125]: I am not sure but change it.

File: docs/dietpi_references.md (hunk 3 above)
Comment (StephanStS) [reply docs/dietpi_references.md:15]: Yes, I took the original titles.

File: docs/dietpi_references.md (hunk 2 above)
Comment (StephanStS) [reply docs/dietpi_references.md:8]: Ok, changed.

**Pull request merged**